fixed.vars

```diff
@@ -5,17 +5,17 @@ LODESTAR_FIXED_VARS="--dataDir /data/lodestar --execution.urls http://127.0.0.1:
 LODESTAR_VAL_FIXED_VARS="--dataDir /data/lodestar"
 
 # Data dir will be mounted in /data/nethermind
-NETHERMIND_FIXED_VARS="--datadir /data/nethermind --Network.DiscoveryPort=30303 --Network.P2PPort=30303 --Merge.Enabled=true --Init.DiagnosticMode=None --JsonRpc.Enabled=true --JsonRpc.Host=0.0.0.0 --JsonRpc.AdditionalRpcUrls \"http://localhost:8545|http;ws|net;eth;subscribe;engine;web3;client|no-auth,http://localhost:8551|http;ws|net;eth;subscribe;engine;web3;client\" --JsonRpc.JwtSecretFile /data/jwtsecret"
+NETHERMIND_FIXED_VARS="--datadir /data/nethermind --Sync.SnapSync=true --Merge.Enabled=true --Init.DiagnosticMode=None --JsonRpc.Enabled=true --JsonRpc.Host=0.0.0.0 --JsonRpc.AdditionalRpcUrls \"http://localhost:8551|http;ws|net;eth;subscribe;engine;web3;client\" --JsonRpc.JwtSecretFile /data/jwtsecret"
 
 # Data dir will be mounted in /data/geth
-GETH_FIXED_VARS="--datadir /data/geth --authrpc.jwtsecret /data/jwtsecret --http --http.api engine,net,eth,web3 --http.port 8545 --allow-insecure-unlock --http.addr 0.0.0.0 --http.corsdomain \"*\" --http.vhosts \"*\" --authrpc.port=8551"
+GETH_FIXED_VARS="--datadir /data/geth --authrpc.jwtsecret /data/jwtsecret --http --authrpc.addr 0.0.0.0 --authrpc.corsdomain \"*\" --authrpc.vhosts \"*\" --authrpc.addr 0.0.0.0 --authrpc.vhosts \"*\" --authrpc.port=8551"
 
 # Data dir will be mounted in /data/ethereumjs
-ETHEREUMJS_FIXED_VARS="--datadir /data/ethereumjs --gethGenesis /config/genesis.json --jwt-secret /data/jwtsecret --saveReceipts --rpc --rpcport 8545 --ws --rpcEngine --rpcEnginePort=8551 --rpcDebug --loglevel=debug"
+ETHEREUMJS_FIXED_VARS="--datadir /data/ethereumjs --jwt-secret /data/jwtsecret --saveReceipts --rpcEngine --rpcEnginePort=8551 --rpcDebug"
 
 # Data dir will be mounted in /data/besu
 BESU_FIXED_VARS="--data-path=/data/besu --engine-jwt-secret=/data/jwtsecret --rpc-http-enabled=true --rpc-http-api=ADMIN,CLIQUE,MINER,ETH,NET,DEBUG,TXPOOL,TRACE --rpc-http-host=0.0.0.0 --rpc-http-port=8545 --engine-rpc-port=8551 --rpc-http-cors-origins=\"*\" --host-allowlist=\"*\" --engine-host-allowlist=\"*\" --p2p-enabled=true --engine-jwt-enabled=true"
 
 # Data dir will be mounted in /data/erigon, private api addr has been switched off because it collides
 # with lodestar prometheus endpoint. If required make adjustments accordingly
-ERIGON_FIXED_VARS="--datadir=/data/erigon --private.api.addr \"\" --http --http.api \"admin,engine,net,eth\" --http.port=8545 --http.addr 0.0.0.0 --http.corsdomain \"*\" --ws --engine.port=8551 --engine.addr=0.0.0.0 --authrpc.jwtsecret=/data/jwtsecret --allow-insecure-unlock --batchSize=32m"
\ No newline at end of file
+ERIGON_FIXED_VARS="--datadir=/data/erigon --private.api.addr \"\" --http --engine.port=8551 --engine.addr=0.0.0.0 --authrpc.jwtsecret=/data/jwtsecret --batchSize=32m"
```

Comment:
`--engine.port=8551` and `--engine.addr=0.0.0.0` is now `--authrpc.port=8551` and `--authrpc.addr=0.0.0.0`